.circleci/config.yml

```diff
@@ -831,6 +831,64 @@ jobs:
           path: tmp/capybara
           destination: capybara
 
+  check_production_gemfile:
+    parameters:
+      executor:
+        type: string
+        default: builder-ruby25
+    executor:
+      name: << parameters.executor >>
+      database: mysql
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "d8:d6:e4:53:b3:0e:22:76:01:e0:27:90:a6:c3:fa:df"
+      - checkout-with-submodules
+      - run:
+          name: Keep production gemfile up to date
+          command: |
+            echo "keeping production gemfiles up to date"
+            git config user.email "3scale@redhat.com"
+            git config user.name "3scale"
+            make bundle
+            FILES_CHANGED=$(git status --porcelain | awk '{print $2}')
+            [[ -z "$FILES_CHANGED" ]] && exit 0
+            if [[ $FILES_CHANGED = "Gemfile.prod.lock"  ]]; then
+              git add Gemfile.prod.lock
+              git commit -m "Update Gemfile.prod.lock"
+              git push origin $(git rev-parse --abbrev-ref HEAD)
+            else
+              echo "More than Gemfile.prod.lock file was changed"
+              curl -X POST -H 'Content-type: application/json' \
+              --data \
+              "{ \
+                \"attachments\": [ \
+                { \
+                  \"fallback\": \"Build failed on $CIRCLE_BRANCH, for $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME.\", \
```

Comment:
CIRCLE_CI_BRANCH is always master right?

Reply:
yeah!